src/js/components/sharedComponents/stickyHeader/StickyHeader.jsx

```diff
@@ -6,12 +6,24 @@
 import React, { useState, useRef, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { throttle } from 'lodash';
+import Cookies from 'js-cookie';
 
 const propTypes = {
     children: PropTypes.node
 };
 
 export const stickyHeaderHeight = 66;
+export const globalBannerHeight = 90;
+export const globalCovidBannerCookie = 'usaspending_covid_release';
+
+export const getStickyBreakPointForSidebar = () => {
```

Comment:
This cookie should probably be the input.

Reply:
Should this also be in a helper file? Perhaps not just wondering.

Reply:
Yeah, the data points above and this function should be moved to a helper and/or datamapping file. I just put it in here since someone put useDynamicStickyClass in this file as well.